scripts/error_codes.py

```diff
@@ -36,24 +36,26 @@ def in_comment(source, pos):
     return slash_star_pos > star_slash_pos
 
 
-def find_ids_in_source_file(file_name, ids):
+def find_ids_in_source_file(file_name, id_to_file_names):
```

Comment:
You're adding doc strings to other functions, but in most it's missing. Any reason for that?

I'm trying to review your PR and reading about what your intentions are (instead of just git diff) would be really helpful. :)

Reply:
> reading about what your intentions are (instead of just git diff) would be really helpful.

The main goal was to print file names which contain uncovered error codes, along with these error codes. It simplifies covering similar errors together, in one step.

And refactoring is a persistent temptation :).

> You're adding doc strings to other functions, but in most it's missing. Any reason for that?

I was adding some comments, and if these comments were related to the entire function, I used a docstring. I did not have an intention to comment every function :).